CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 0.5.0 (2022-06-23)
+
+### Features
```

Comment:
### Breaking

- change "fromSubAccountId: string" param to "fromSubAccount: number[]"

### Features

- Remove `payload`, `payloadBytes` and `nnsFunctionName` fields from proposal of type ExecuteNnsFunction action (moved to nns-dapp)
- use Candid interface by default for ledger canister and proto for hardware wallet
- new function `transactionFee` to query the current transaction fee

Reply:
So is this all good now?